src/styles/calendar.scss

```diff
@@ -0,0 +1,53 @@
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  width: 7 * $square100 + 6 * $space-between +20px;
+
+  padding: 10px;
+  gap: $space-between;
+
+  
+
```

Comment:
Let's display the calendar in the center of the screen. 
For example, you can create a separate block for the body and consider using flex on it with align-items: center.